pycbc/distributions/sky_location.py

```diff
@@ -31,5 +33,47 @@ class UniformSky(angular.UniformSolidAngle):
     _default_polar_angle = 'dec'
     _default_azimuthal_angle = 'ra'
 
+class FisherDist():
+    """A distribution that returns a random (ra, dec) angle drawn from the
+    Fisher distribution. Assume that the concentration parameter (kappa)
+    is large so that we can use a Rayleigh distribution about the north
+    pole and rotate it to be centered at the (ra, dec) coordinate mu.
 
-__all__ = ['UniformSky']
+    Assume kappa = 1 / sigma**2
+
+    As in UniformSky, the polar angle varies from pi/2 to-pi/2
+    and dec varies from 0 to 2pi.
+
+    References:
+      * http://en.wikipedia.org/wiki/Von_Mises-Fisher_distribution
+      * http://arxiv.org/pdf/0902.0737v1 (states the Rayleigh limit)
+    """
+    name = 'fisher_dist'
+    _polardistcls = angular.CosAngle
+    _default_polar_angle = 'dec'
+    _default_azimuthal_angle = 'ra'
+
+    def __init__(self, mu, kappa, if_radians=True):
+        self.kappa = kappa
+        if if_radians is True:
+            self.mu = (mu[1], mu[0])
+        elif if_radians is False:
+            self.mu = numpy.array(numpy.deg2rad([mu[1], mu[0]]))
+        else:
+            raise ValueError("You should give either 'True' or 'False' "
+                             "(for whether the angles are in radians or "
+                             "degrees respectively)")
```

Comment:
In the spirit of keeping things as simple as necessary, I would name the parameter just `radians`, and implement the code as follows:
```suggestion
        if radians:
            self.mu = (mu[1], mu[0])
        else:
            self.mu = numpy.array(numpy.deg2rad([mu[1], mu[0]]))
```

Reply:
The docstring should also indicate what the `radians` parameter does (see docstrings for other classes for an example).

Reply:
The suggested change is made and additional details regarding radians is provided in the docstring